src/components/ebay-button/index.marko

```diff
@@ -40,6 +40,7 @@ $ {
     var fixedHeightClass =
         input.fixedHeight &&
         (sizeClass ? sizeClass + "-fixed-height" : baseClass + "--fixed-height");
+    var destructiveClass = input.destructive ? baseClass + "--destructive" : "";
```

Comment:
We tend to prefer to to have a "enum" type value for things that might have multiple values. 
So maybe 
`variant="destructive"` or `variant="standard"` (default value)
Because we might add different type.

Reply:
So change to this?:

`var variantClass = input.variant ? baseClass + "--" + variant : "";`

If so, I imagine we would need to also change the destructive toggle (bool) in `button.stories.js` from toggle to possibly a combobox with (for now) dropdown option of "destructive?"

Reply:
Yes, that's correct. 
See how priority is. 

Usually we have a default like `none` or `standard`.

Reply:
@agliga , gotcha. Ok, checked in the tweaks with commit verification.